lib/index.js

```diff
@@ -77,13 +87,46 @@ export default {
     this.subscriptions.dispose();
   },
 
+  fixJob(editor) {
+    let correctScope = false;
+
+    if (editor && editor.buffer && editor.buffer.file && editor.buffer.file.path) {
```

Comment:
Use [Workspace::isTextEditor](https://atom.io/docs/api/v1.26.1/Workspace#instance-isTextEditor) and [TextEditor::getPath](https://atom.io/docs/api/v1.26.1/TextEditor#instance-getPath).